app/controllers/finance/ecf/statements_controller.rb

```diff
@@ -6,8 +6,12 @@ class StatementsController < BaseController
       def show
         @ecf_lead_provider = lead_provider_scope.find(params[:payment_breakdown_id])
         @statement = @ecf_lead_provider.statements.find(params[:id])
-        @calculator = StatementCalculator.new(statement: @statement)
+        @calculator = ECT::StatementCalculator.new(statement: @statement)
         set_important_message(title: t("finance.statements.payment_authorisations.banner.title"), content: t("finance.statements.payment_authorisations.banner.content", statement_marked_as_paid_at: @statement.marked_as_paid_at.strftime("%-I:%M%P on %-e %b %Y"))) if authorising_for_payment_banner_visible?(@statement)
+
+        if @statement.cohort.mentor_funding?
```

Comment:
I think we have a ticket for the FE finance calculators, can we please leave out the changes from this one?

Reply:
Ah yeah tks @cwrw I committed those ones.. forgot to remove it before pusing.. I had added just to be able to see calculators results in the page. tks